src/ngMessages/messages.js

```diff
@@ -552,22 +552,32 @@ angular.module('ngMessages', [])
         $templateRequest(src).then(function(html) {
           if ($scope.$$destroyed) return;
 
-          $compile(html)($scope, function(contents) {
-            element.after(contents);
-
-            // the anchor is placed for debugging purposes
-            var comment = $compile.$$createComment ?
-                $compile.$$createComment('ngMessagesInclude', src) :
-                $document[0].createComment(' ngMessagesInclude: ' + src + ' ');
-            var anchor = jqLite(comment);
-            element.after(anchor);
-
-            // we don't want to pollute the DOM anymore by keeping an empty directive element
-            element.remove();
-          });
+          if (isString(html) && !html.trim()) {
```

Comment:
The `isString()` is needed in case the request returns some non-string data. This is normally not the case, but can happen (especially in tests).